pytentiostat/config_reader.py

```diff
@@ -81,29 +158,108 @@ def get_cv_params(config_data):
 
 
 def get_exp_type(config_data):
+    """
+    Returns the type of experiment from the config data.
+
+    Parameters
+    ----------
+    config_data : dict
+        The configuration data
+
+    Returns
+    -------
+    exp_type : str
+        The type of experiment to be run.
+    """
+    
     exp_type = config_data["general_parameters"]["experiment_type"]
 
     return exp_type
 
 
 def get_exp_time(config_data):
```

Comment:
I wonder if this function should be called ``get_experiment_duration``?

Reply:
That is more descriptive. Now changed